src/couch_replicator_tasks.erl

```diff
@@ -0,0 +1,52 @@
+-module(couch_replicator_tasks).
+
+-export([all/0]).
+
+-include_lib("couch/include/couch_db.hrl").
+-include("couch_replicator.hrl").
+-include("couch_replicator_api_wrap.hrl").
+
+
+all() ->
```

Comment:
If it is just one function, is it better to just have scheduler know how to translate its own state to ejson, that way we keep the job record internals private to that one modules?

Unless the intent was that this module will grow more function in immediate future and and we won't want scheduler to be bigger than it is. I am ok with that then.

Another thing to think about, maybe make `couch_replicator.erl` the external API module. So calls to get scheduler task from outside the app should be proxies through there perhaps.